io-apps/sysbench/sysbench-prepare.dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM severalnines/sysbench
```

Comment:
Can we have just one container with both the scripts and call either verify.sh or prepare.sh from the pod?

Reply:
We don't even need a separate container IMO. We should just use the sysbench container and put the command in the pod spec. That way we don't need to maintain this extra container

Reply:
Done, I've used only vanila sysbench and mysql client images.